In [1]:
import pandas as pd
import numpy as np

In [3]:
df=pd.read_csv('train.csv')

**Basic Data Overview**

In [6]:
df.head(20)

id keyword location                                               text  \
0    1     NaN      NaN  Our Deeds are the Reason of this #earthquake M...   
1    4     NaN      NaN             Forest fire near La Ronge Sask. Canada   
2    5     NaN      NaN  All residents asked to 'shelter in place' are ...   
3    6     NaN      NaN  13,000 people receive #wildfires evacuation or...   
4    7     NaN      NaN  Just got sent this photo from Ruby #Alaska as ...   
5    8     NaN      NaN  #RockyFire Update => California Hwy. 20 closed...   
6   10     NaN      NaN  #flood #disaster Heavy rain causes flash flood...   
7   13     NaN      NaN  I'm on top of the hill and I can see a fire in...   
8   14     NaN      NaN  There's an emergency evacuation happening now ...   
9   15     NaN      NaN  I'm afraid that the tornado is coming to our a...   
10  16     NaN      NaN        Three people died from the heat wave so far   
11  17     NaN      NaN  Haha South Tampa is getting flooded hah- WAIT ...   
12  18     NaN      NaN  #raining #flooding #Florida #TampaBay #Tampa 1...   
13  19     NaN      NaN            #Flood in Bago Myanmar #We arrived Bago   
14  20     NaN      NaN  Damage to school bus on 80 in multi car crash ...   
15  23     NaN      NaN                                     What's up man?   
16  24     NaN      NaN                                      I love fruits   
17  25     NaN      NaN                                   Summer is lovely   
18  26     NaN      NaN                                  My car is so fast   
19  28     NaN      NaN                       What a goooooooaaaaaal!!!!!!   

    target  
0        1  
1        1  
2        1  
3        1  
4        1  
5        1  
6        1  
7        1  
8        1  
9        1  
10       1  
11       1  
12       1  
13       1  
14       1  
15       0  
16       0  
17       0  
18       0  
19       0

In [7]:
df.shape

(7613, 5)

In [8]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 7613 entries, 0 to 7612
Data columns (total 5 columns):
 #   Column    Non-Null Count  Dtype 
---  ------    --------------  ----- 
 0   id        7613 non-null   int64 
 1   keyword   7552 non-null   object
 2   location  5080 non-null   object
 3   text      7613 non-null   object
 4   target    7613 non-null   int64 
dtypes: int64(2), object(3)
memory usage: 297.5+ KB


* We have 5 columns, many missing values in keyword and location
* There are only 7613 rows 
* there are no Nan values in text colmn 